dashboard/src/pages/CreateEventForm.vue

```diff
@@ -4,84 +4,120 @@
       <ChapterHeader :chapter="chapter" />
     </div>
     <div class="flex flex-col md:flex-row gap-2 justify-between mt-6 pb-2 border-b">
-      <div class="text-xl font-semibold">Create Event</div>
-      <Button label="Create" :variant="'solid'" size="md" @click="createEvent" />
+      <div class="prose">
+        <h2>Create Event</h2>
+      </div>
     </div>
     <div v-if="eventTypeOptions.data">
-      <div class="flex flex-col gap-3 my-6">
-        <div class="font-semibold text-gray-800 border-b-2 pb-2">Details</div>
-        <div class="p-2 my-1 grid sm:grid-cols-1 md:grid-cols-2 gap-x-4 gap-y-6">
-          <FormControl
-            v-model="temp_event.event_name"
-            :type="'text'"
-            size="md"
-            label="Event Name"
-          />
-          <FormControl
-            v-model="temp_event.event_permalink"
-            :type="'text'"
-            size="md"
-            label="Event Permalink"
-            description="Only enter the event endpoint. Events will be rendered at event/<event_permalink>"
-          />
-          <FormControl
-            v-model="temp_event.status"
-            :type="'select'"
-            :options="[
-              {
-                label: 'Draft',
-                value: 'Draft',
-              },
-              {
-                label: 'Live',
-                value: 'Live',
-              },
-              {
-                label: 'Concluded',
-                value: 'Concluded',
-              },
-              {
-                label: 'Cancelled',
-                value: 'Cancelled',
-              },
-            ]"
-            size="md"
-            label="Event Status"
-            description="Current status of the event."
-          />
-          <FormControl
-            v-model="temp_event.event_type"
-            :type="'select'"
-            :options="eventTypeOptions.data"
-            size="md"
-            label="Event Type"
-          />
-          <TextEditor
-            label="Event Description"
-            class="col-span-2"
-            placeholder="Write an event description"
-            :model-value="temp_event.event_description"
-            @update:model-value="($event) => (temp_event.event_description = $event)"
-          />
+      <div class="flex flex-col gap-5 my-6">
```

Comment:
I don't see it in the UI - but I think it would also be better to add a red asterisk or so - to mark fields that are important. 

That way the occurrence of MandatoryFields Error is actually minimised.

Reply:
I think just an asterisk would suffice, as other forms too has an asterisk with the label (gray not red). Gray asterisk is there in this form as well